src/build.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LearderBoard</title>
+  </head>
+  <body>
+    <h1 class="hello" id="ip">LearderBoard</h1>
+    <section id="container">
+      <div id="list">
+        <div id="L-con">
+          <h2>
+            Recent scores 
+          </h2>
+          <button>
+            Refresh
+          </button>
+        </div>
+<ul>
+  <li>
+   <p>Name: 100</p>
+  </li>
+  <li>
+   <p>Name: 100</p>
+  </li>
+  <li>
+    <p>Name: 100</p>
+  </li>
+  <li>
+    <p>Name: 100</p>
+  </li>
+  <li>
+    <p>Name: 100</p>
+  </li>
+  <li>
+    <p>Name: 100</p>
+  </li>
+  <li>
+    <p>Name: 100</p>
+  </li>
+</ul>
```

Comment:
- [ ] This section should be generated using JS DOM, please do so to comply with the project requirements. Don't forget about ES6 syntax.